src/python/grpcio_tests/tests_aio/unit/client_unary_unary_interceptor_test.py

```diff
@@ -224,6 +224,51 @@ async def intercept_unary_unary(self, continuation,
             self.assertEqual(await interceptor.calls[1].code(),
                              grpc.StatusCode.OK)
 
+    async def test_retry_with_multiple_interceptors(self):
+
+        class RetryInterceptor(aio.UnaryUnaryClientInterceptor):
+            """a Retry Interceptor which calls another interceptor"""
```

Comment:
Technically speaking, this isn't true. This interceptor only calls another interceptor because of the `interceptors` kwarg to `aio.secure_channel` below. I think it would be more explanatory to move a comment like this down there.

Reply:
That's right, I'll move this down.